app/controllers/forgot_passwords_controller.rb

```diff
@@ -0,0 +1,15 @@
+class ForgotPasswordsController < ApplicationController
+
+
+def create
+  user = User.where(email: params[:email]).first
+  if user
+    AppMailer.send_forgot_password(user).deliver
+    redirect_to forgot_password_confirmation_path
+  else
+    flash[:danger] = params[:email].blank? ? "Email cannot be blank." : "There is no user with that email in the system."
```

Comment:
Since this is so line, I would recommend changing this to a five line `if`/`else`/`end`. You can still use the assignment operator.

```ruby
flash[:danger] = if params[:email].blank?
                            "Email cannot be blank."
                          else
                            "There is no user with that email in the system."
                          end
```